src/utilities/validators/parameter-validator.ts

```diff
@@ -64,7 +65,12 @@ class ParameterValidator extends BaseValidator {
     example: Json,
     path: string[],
   ): void {
-    if (parameterHasSchema(parameter) && !parameterHasContent(parameter)) {
+    if (parameter.example && parameter.examples) {
+      this._failures = [...this._failures, new InvalidParameterExample(path)];
```

Comment:
part of the ticket mentions out putting 'A helpful message about the use of the 'default' parameter group'. I think the best way to do this would be to add a warning here as well, but could also see it being added to the failure. I wonder if we need to introduce the 'verbose' mode here, and add additional output to a failure that only gets output in verbose mode?